static/app/views/performance/newTraceDetails/traceHeader/index.tsx

```diff
@@ -297,108 +181,85 @@ export function TraceMetaDataHeader(props: TraceMetadataHeaderProps) {
     props.tree.type === 'error';
 
   if (isLoading || isError) {
-    return <PlaceHolder organization={props.organization} />;
+    return (
+      <PlaceHolder
+        organization={props.organization}
+        project={undefined}
+        traceSlug={props.traceSlug}
+      />
+    );
   }
 
-  const representativeEvent = getRepresentativeEvent(props.tree);
+  const representativeEvent = getRepresentativeEvent(props.tree, props.logs);
+  const project = representativeEvent
+    ? projects.find(p => {
+        if (OurLogKnownFieldKey.PROJECT_ID in representativeEvent) {
```

Comment:
nit: I think it makes sense to move this `if` condition outside of the `find` loop. Technically, it's running this check on each `find` iteration, when you actually only need to do it once.

Reply:
This one, I'm just keeping as is